libraries/AP_IOMCU/AP_IOMCU.cpp

```diff
@@ -65,7 +65,6 @@ void AP_IOMCU::init(void)
 {
     // uart runs at 1.5MBit
     uart.begin(1500*1000, 128, 128);
-    uart.set_blocking_writes(true);
```

Comment:
Seems like a large behavior change for what IIRC is a very timing sensitive protocol?

Reply:
This isn't a change in behaviour; Hott telemetry (and indeed, everywhere that sets blocking writes) also marks the serial port as "unbuffered".

Weird thing is that Hott telemetry on master retries sending the byte itself even 'though it marks the port as blocking.